Gaijinizer patch/Map042.txt

```diff
@@ -557,104 +612,111 @@ I don't know so much. 'Human' is broken.#MTLed
 舌の数だけ分散しただけで、
 そもそも人間相手だと、そんなに魔力は要らないと…
 > CONTEXT: Map042/events/5/pages/1/244/Dialogue/kazuya
-In other words, originally you have higher magic
-power. Just by the number of tongues,
-If you're a human opponent in the first place,
-you don't need so much magic ...#MTLed
+I- in other words, you originally had more magic
+power than them. But even though your magic is now
+dispersed across all your tongues, you don't need
+to use much magic against humans...#RonIffy
 > END STRING
 
 > BEGIN STRING
 …………\\N[1]、よく分かってる。
 
 ……スキ。
 > CONTEXT: Map042/events/5/pages/1/250/Dialogue/enemy2
-............ \\N[1], I understand very well.
-……Love.#MTLed
+... \\N[1], you really... get me...
+
+...I love you.#Ron
 > END STRING
 
 > BEGIN STRING
 （な、なんで勝ち誇ったような顔なんだろう…？）
 > CONTEXT: Map042/events/5/pages/1/254/Dialogue/kazuya
-(Why, why is it a face that won?)#MTLed
+(W- why does she look so triumphant?)#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]。
 
 わたし　バーゲスト　どっち　すきか？
 > CONTEXT: Map042/events/5/pages/1/257/Dialogue/enemy2
-\\N[1]. I like barghest which one?#MTLed
+\\N[1].
+
+Barghests... or me... which... do you...like
+more...?#Ron
 > END STRING
 
 > BEGIN STRING
 （ダイレクトな２択で聞いてきたなー！！）
 > CONTEXT: Map042/events/5/pages/1/261/Dialogue/kazuya
-(Have heard it in a direct 2 options !!)#MTLed
+(Uh oh, she's only giving me the two options!!)#Ron
 > END STRING
 
 > BEGIN STRING
 え、えっと……
 
 そもそも種族がちがうし、どっちにも良いとこ……ろ…-
 > CONTEXT: Map042/events/5/pages/1/263/Dialogue/kazuya
-Let me see……. In the first place, the races are
-different, and both are good.#MTLed
+L- let me see... Both races have their good points
+so I don't think... I could...#Ron
 > END STRING
 
 > BEGIN STRING
 ……が…-
 > CONTEXT: Map042/events/5/pages/1/267/Dialogue/kazuya
-……But…-#MTLed
+... BUT...#Ron
 > END STRING
 
 > BEGIN STRING
 も、もちろん………！君さ！エンプーサ！
 > CONTEXT: Map042/events/5/pages/1/271/Dialogue/kazuya
-Well, of course......! You are! Empuse!#MTLed
+N- naturally I pick you, Empuse!#Ron
 > END STRING
 
 > BEGIN STRING
 （あんな風に鎌を振り上げられたら
 
 　そうとしか言えないじゃんか……！！）
 > CONTEXT: Map042/events/5/pages/1/275/Dialogue/kazuya
-(If you can raise your ax like that
-I can only say so.........!!)#MTLed
+(Of course I'm gonna pick her if she raises
+her sickles like that...!!)#Ron
 > END STRING
 
 > BEGIN STRING
 …………\\N[1]　つぎ　なに　はなし　する？
 
 交尾　でも　いいぞ？
 > CONTEXT: Map042/events/5/pages/1/279/Dialogue/enemy2
-............ \\N[1] What would you like to do
-next? Mating But is it okay?#MTLed
+... \\N[1]... what would... you like to... talk
+about... next...? We can... mate... if you...
+want to...#Ron
 > END STRING
 
 > BEGIN STRING
 （あ、でも、具体的なこととか深い部分の話に
 
 　持っていこうとはしないんだな………？）
 > CONTEXT: Map042/events/5/pages/1/283/Dialogue/kazuya
-(Oh, but in the specifics and in-depth stories
-You don't want to bring it......? )#MTLed
+(Ah, but maybe I shouldn't try to get into any
+really deep topics of conversation...)#Ron
```

Comment:
If we take these next few sets of lines together, we can make them flow a bit better with a few changes.
I'd add "her" after get here.